spinn_front_end_common/utilities/utility_objs/executable_type.py

```diff
@@ -0,0 +1,62 @@
+from enum import Enum
+from spinnman.model.enums import CPUState
+
+
+class ExecutableType(Enum):
+    """ supports starting of different types of executables
+    """
+
+    RUNNING = (
+        0,
+        [CPUState.RUNNING],
+        [CPUState.FINISHED],
+        False,
+        "Runs immediately without waiting for barrier and then exits")
+
+    SYNC = (
+        1,
+        [CPUState.SYNC0],
```

Comment:
I think this needs a FINISHED as well, in case it finishes before you look

Reply:
yep, dead on. i put them in, then took them out, convincing myself it was safe. but nah ya right

spurred from donals comment of 1 state. lol

Reply:
rubbered